src/action/transaction.js

```diff
@@ -63,7 +63,7 @@ class TransactionAction {
         amount: transaction.amount,
         fee: transaction.totalFees,
         confirmations: transaction.numConfirmations,
-        status: transaction.numConfirmations < 1 ? 'unconfirmed' : 'confirmed',
+        status: transaction.numConfirmations < 3 ? 'unconfirmed' : 'confirmed',
```

Comment:
Note: in the past, we've gotten some feedback when people are able to send payments through channels that are displayed as `pending open`.

Reply:
Mmmh. That's wierd. Let's watch out for that. This change only affects the transaction list status.